aws-opensearchserverless-lifecyclepolicy/.rpdk-config

```diff
@@ -0,0 +1,29 @@
+{
+    "artifact_type": "RESOURCE",
+    "typeName": "AWS::OpenSearchServerless::LifecyclePolicy",
+    "language": "java",
+    "runtime": "java8",
```

Comment:
Can you double check if correct java version is used here?

Reply:
Checked. All the CFN resources are using java8